arbitrum-docs/partials/_migrate-nft-collections-partial.md

```diff
@@ -0,0 +1,90 @@
+This how-to is for **users** and **developers** who want to migrate (or "bridge") existing NFT collections onto Arbitrum.
+
+## Why migrate NFTs to Arbitrum?
+
+There are two main reasons to migrate NFTs to Arbitrum:
+
+ 1. **Lower fees**. Arbitrum's gas fees are significantly lower than Ethereum's, meaning that you can mint, transfer, and trade NFTs without having to pay as much.
+ 2. **Fast transactions**. From the end-user's perspective, transactions on Arbitrum's rollup chains are processed more quickly than they are on Ethereum mainnet. This lets developers build NFT experiences that feel just about as fast as traditional web apps.
+
+<!-- todo: when shouldn't you? Risks to call out? -->
+
+:::caution
+
+It's very important to understand that **the [Arbitrum Bridge](https://bridge.arbitrum.io/) does not support ERC-721 or ERC-1155 token transfers**; it supports transfers of **ERC-20 tokens** and **plain-old ETH** between Ethereum and Arbitrum's two chains.
+
+:::
+
+## Tokens and bridges
+
+Before we begin, let's align on the various types of tokens that can be used as NFTs, and how they can be migrated to Arbitrum.
```

Comment:
@amarrazza a sanity check here would be wonderful 🙏

Reply:
I like the addition of the table here but would remove the ERC-20 row as there isn't anyone using it as an NFT-token type